lib/ext/utils/base.ex

```diff
@@ -102,16 +216,49 @@ defmodule Ext.Utils.Base do
     end)
   end
 
+  @doc ~S"""
+    Convert map to keyword_list
+
+    ## Examples
+
+      iex> Ext.Utils.Base.to_keyword_list(%{first: "value1", second: "value2"})
+      [first: :value1, second: :value2]
+  """
+
   def to_keyword_list(data) do
-    Enum.reduce(data, [], fn {key, value}, acc -> [{to_atom(key), to_atom(value)} | acc] end)
+    Enum.reduce(data, [], fn {key, value}, acc -> acc ++ [{to_atom(key), to_atom(value)}] end)
```

Comment:
I'm not sure this function is useful. Could we check we it's used?

`%{first: "value1", second: "value2"} |> Enum.to_list()` has almost the same functionality